src/box/memtx_tree.cc

```diff
@@ -951,6 +991,91 @@ memtx_tree_index_get_internal(struct index *base, const char *key,
 	return 0;
 }
 
+/**
+ * Implementation of iterator position for general and multikey indexes.
+ */
+template <bool USE_HINT, bool IS_MULTIKEY>
+static inline int
+tree_iterator_position(struct iterator *it, const char **pos, uint32_t *size)
```

Comment:
Nit: this function is called by pointer so `inline` doesn't make any sense. Please remove.